### PR TITLE
Add eks doc

### PR DESCRIPTION
**Pull request by tei-k**
# Description

https://scalar-labs.atlassian.net/browse/DLT-7839

### CODE REVIEW

File: docs/EKSScalarTerraformDeploymentGuide.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# How to create AWS EKS with scalar-terraform
+
+This document shows how to create AWS Kubernetes Service (EKS) along with a VPC, a bastion host, and database resources using Terraform scripts.
```
Comment (feeblefakie) [inline docs/EKSScalarTerraformDeploymentGuide.md:3]:
```suggestion
This document shows how to create a Kubernetes cluster in Amazon Elastic Kubernetes Service (EKS) along with a VPC, a bastion host, and database resources using Terraform scripts.
```

**Review by feeblefakie — COMMENTED** [review]
Left some suggestions.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thanks!

File: docs/EKSScalarTerraformDeploymentGuide.md (hunk 1 above)
Comment (tei-k) [reply docs/EKSScalarTerraformDeploymentGuide.md]: Thanks! Fixed in c04e773.

File: docs/AKSScalarTerraformDeploymentGuide.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# How to create Azure AKS with scalar-terraform
+# How to Create a Kubernetes cluster in Azure AKS with scalar-terraform
```
Comment (ymorimo) [inline docs/AKSScalarTerraformDeploymentGuide.md:1]:
```suggestion
# How to create a Kubernetes cluster in Azure AKS with scalar-terraform
```

**Review by ymorimo — APPROVED** [review]
LGTM except for one minor suggestion about capitalization.

File: docs/AKSScalarTerraformDeploymentGuide.md (hunk 2 above)
Comment (tei-k) [reply docs/AKSScalarTerraformDeploymentGuide.md]: Thanks! fixed in 344e4f3.

**Pull request merged**